src/models/jwtPermissions/index.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+const { JWTVerifier } = require('@storyous/storyous-jwt');
+const fs = require('fs');
+const path = require('path');
+
+// eslint-disable-next-line import/no-unresolved
+const { permissionHelper, fetch } = require('../../index');
+const { NotSufficientPermissions } = require('./customErrors');
+const { isProduction, isTesting } = require('../../config');
+
+const publicKeys = {};
+let _publicKeyUrl;
+let keyName;
+if (isProduction()) { _publicKeyUrl = 'http://publickey.storyous.com'; keyName = 'publicProduction'; } else
+if (isTesting()) {
+    _publicKeyUrl = 'http://127.0.0.1:3010/getPublicKey'; keyName = 'publicTesting';
+} else { _publicKeyUrl = 'http://publickey.storyous.com'; keyName = 'publicTest'; }
+const publicKeyPath = path.join(__dirname, '..', '..', '..', 'publicKeys', keyName);
+
+/**
+ *
+ * @param {null|string}authorization
+ * @returns {string}
+ */
+const parseAuthorization = (authorization) => {
+    if (!authorization) { return ''; }
+    const parsedAuthorization = authorization.split(' ').filter((i) => i !== ' ');
```

Comment:
You should check for right format, which is `Bearer {token}`.
`SomeOtherString {token}` should be considered invalid

Reply:
done

Reply:
@tymak - I dont see it done